### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by dusking**

### CODE REVIEW

File: scripts/nginx/preconfigure_nginx.py [hunk 1]
```diff
@@ -17,23 +17,21 @@
 
 def _set_nginx_config_file(host_ip):
     config_file = os.path.join(os.path.expanduser('~'), 'djangosample/src/mysite/nginx.conf')
+    project_root_dir = os.path.join(os.path.expanduser('~'), 'djangosample/src')
```
Comment (dankilman) [inline scripts/nginx/preconfigure_nginx.py]: can be simplified. no need for os.path.join
`os.path.expanduser('~/djangosample/src')`
same goes for `config_file`

Comment (dankilman) [inline scripts/nginx/preconfigure_nginx.py]: although i would generally first calculate project_root_dir
and then have config_file relative to it using os.path.join

File: scripts/nginx/preconfigure_nginx.py [hunk 2]
```diff
@@ -17,23 +17,21 @@
...
     params = ctx.source.node.properties.copy()
-    params.update({'host_ip': host_ip})
+    params.update({'host_ip': host_ip,
+                   'project_root_dir': project_root_dir})
     sampleutils.download_resource_and_render(source='config/nginx/nginx.conf.template',
```
Comment (dankilman) [inline scripts/nginx/preconfigure_nginx.py]: again, why are you using sampleutils?

File: scripts/nginx/preconfigure_nginx.py (hunk 1 above)
Comment (dusking) [inline scripts/nginx/preconfigure_nginx.py]: need the os.path.join, since nginx.conf need absolute path (no ~)

Comment (dankilman) [inline scripts/nginx/preconfigure_nginx.py]: `os.path.join(os.path.expanduser('~'), 'djangosample/src/mysite/nginx.conf') == os.path.expanduser('~/djangosample/src')`

**Pull request merged**